ui/components/multichain/notifications-tag-counter/notifications-tag-counter.tsx

```diff
@@ -109,18 +109,17 @@ export const NotificationsTagCounter = ({
     <Box
       backgroundColor={BackgroundColor.errorDefault}
       borderStyle={BorderStyle.none}
-      borderRadius={BorderRadius.MD}
+      borderRadius={BorderRadius.LG}
       paddingTop={0}
       paddingBottom={0}
-      paddingLeft={2}
-      paddingRight={2}
+      className="notifications-tag-counter"
     >
       <Text
         color={TextColor.errorInverse}
         variant={TextVariant.bodySm}
```

Comment:
Should this be `TextVariant.bodyXs` instead of setting `font-size:10px`?

Reply:
yep, fixed!